packages/atomic/src/components/common/tabs/tab-popover.pcss

```diff
@@ -0,0 +1,18 @@
+@import '../../../global/global.pcss';
+
+tab-popover {
+  position: absolute;
+}
+
+.visibility-hidden {
+  visibility: hidden;
+}
+
+[part~='overflow-tabs'] {
```

Comment:
Why `~=` and not `=` ?

Reply:
no reason that I can think of. changed to `=`